lottie/src/main/java/com/airbnb/lottie/utils/DefaultLogger.java

```diff
@@ -0,0 +1,46 @@
+package com.airbnb.lottie.utils;
+
+import android.util.Log;
+
+import com.airbnb.lottie.L;
+
+import java.util.HashSet;
+import java.util.Set;
+
+// Default logger
+public class DefaultLogger implements LottieLogger {
```

Comment:
Can you rename this to `LogcatLogger` and also indicate that warnings will only be logged once max in the javadoc?

Reply:
Will do